alchemy/src/sentry/api.ts

```diff
@@ -0,0 +1,102 @@
+/**
+ * Options for Sentry API requests
+ */
+export interface SentryApiOptions {
+  /**
+   * API token to use (overrides environment variable)
+   */
+  apiToken?: string;
+}
+
+/**
+ * Minimal API client using raw fetch
+ */
+export class SentryApi {
+  /** Base URL for API */
+  readonly baseUrl: string;
+
+  /** API token */
+  readonly apiToken: string;
```

Comment:
should this be called `authToken` to match `SENTRY_AUTH_TOKEN` and Sentry (who calls them Auth Tokens)?

Reply:
Resolved via